template/python3-flask/Dockerfile

```diff
@@ -34,13 +34,14 @@ WORKDIR /home/app/function/
 COPY function/requirements.txt	.
 RUN pip install --user -r requirements.txt
 
-WORKDIR /home/app/
-
 #install function code
 USER root
 
-COPY function   function
-RUN chown -R app:app ./
+COPY function/   .
```

Comment:
I wonder why we did and do this part as root?

Reply:
probably need it for chown, i guess